build.sbt

```diff
@@ -51,6 +51,16 @@ def playJsonMimaSettings = Seq(
     case _ => throw new Error("Unable to determine previous version")
   }),
   mimaBinaryIssueFilters ++= Seq(
+    // remove deprecated methods
+    // https://github.com/playframework/play-json/pull/861
+    ProblemFilters.exclude[DirectMissingMethodProblem]("play.api.libs.json.JsBoolean.productArity"),
```

Comment:
Maybe we could bump the mimaPreviousVersion instead ?

Reply:
Bump to what? IMHO it's ok to add those filters here. Thanks!

Reply:
I bumping `mimaPreviousArtifacts` is sufficient, it's much better than adding those filters